packages/commonwealth/server/util/roles.ts

```diff
@@ -66,58 +63,79 @@ export async function getHighestRoleFromCommunityRoles(
   }
 }
 
+function convertToAddressQuery(findOptions: FindOptions) {
+  if (findOptions.where['address_id']) {
+    findOptions.where['id'] = findOptions.where['address_id'];
+    delete findOptions.where['address_id'];
+  }
+
+  return findOptions;
+}
+
 // Server side helpers
 export async function findAllCommunityRolesWithRoleAssignments(
```

Comment:
Wondering if our unit tests cover this.

Reply:
I am not sure how to get good test coverage for this function because the range of inputs is too wide (This is a wrapper on the sequelize query). Instead I just count it as tested due to it being used in the different endpoints which we have integration tests for. I figure this is good since this method should be deprecated (because it is only a shim), and we should slowly start replacing it with direct calls to the db.